lib/bolt/config.rb

```diff
@@ -100,8 +115,8 @@ def update_from_file(data)
         if data['ssh']['private-key']
           self[:transports][:ssh][:key] = data['ssh']['private-key']
         end
-        if data['ssh']['insecure']
-          self[:transports][:ssh][:insecure] = data['ssh']['insecure']
+        if data['ssh'].key?('host_key_check')
```

Comment:
This should be hyphenated rather than underscored, to match `private-key`.

Reply:
fixed